go.mod

```diff
@@ -0,0 +1,132 @@
+module github.com/gardener/gardener
+
+go 1.12
+
+require (
```

Comment:
Is it possible to not include the indirect dependencies here? It's very hard to maintain this way.

Reply:
Currently I am not aware of such way.
Ref https://github.com/golang/go/wiki/Modules#why-does-go-mod-tidy-record-indirect-and-test-dependencies-in-my-gomod